OkienkoStartowe.java

```diff
@@ -0,0 +1,280 @@
+import javafx.application.Application;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.scene.Group;
+import javafx.scene.Scene;
+import javafx.scene.control.*;
+import javafx.scene.image.ImageView;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+import javafx.stage.Stage;
+
+import java.time.LocalDate;
+import java.util.Random;
+
+public class OkienkoStartowe extends Application {
+
+    public static void main(String[] args) {
+        launch(args);
+    }
+
+    @Override
+    public void start(Stage primaryStage) {
+
+        Random rand = new Random();
+
+        Integer Cena = rand.nextInt(150)+50; // losuje cene biletu
+        System.out.println("Cena za 1 bilet: "+Cena);
+
+
+        int los = rand.nextInt(10)+1;  // losuje czy bilety dostepne - 1-7, nie dostepne 8-10
+        System.out.println("Wynik losu dla dat: "+los);
+
+
+        Label label = new Label("Rezerwacja Biletow Lotniczych");
+        Font font = Font.font("Times New Roman", FontWeight.EXTRA_BOLD,28);
+        label.setLayoutX(220);
+        label.setLayoutY(30);
+        label.setFont(font);
+        label.setTextFill(Color.GREEN);
+
+        LocalDate Data =  LocalDate.now();
+        System.out.println(Data);  // == Data.toString()
+
+        Label labeltime = new Label("Dzis jest: "+Data);
+        labeltime.setLayoutX(80);
+        labeltime.setLayoutY(200);
+        labeltime.setTextFill(Color.GREEN);
+
+
+        Label biletTak = new Label("Bilety Dostepne");
+        biletTak.setLayoutX(60);
+        biletTak.setLayoutY(400);
+        biletTak.setTextFill(Color.GREEN);
+        biletTak.setVisible(false);
+
+        RadioButton wObieStrony = new RadioButton("w obie strony");
+        wObieStrony.setLayoutX(50);
+        wObieStrony.setLayoutY(80);
+
+
+        RadioButton wJednaStrone = new RadioButton("w jedna strone");
+        wJednaStrone.setLayoutX(150);
+        wJednaStrone.setLayoutY(80);
+
+
+        ToggleGroup togglegroup = new ToggleGroup();
+        wObieStrony.setToggleGroup(togglegroup);
+        wJednaStrone.setToggleGroup(togglegroup);
+
+
+        Label biletNie = new Label("Bilety Nie dostepne, podaj inne daty");
+        biletNie.setLayoutX(60);
+        biletNie.setLayoutY(450);
+        biletNie.setTextFill(Color.RED);
+        biletNie.setVisible(false);
+
+        TextField dataWylotu = new TextField();
+        dataWylotu.setPromptText("DATA WYLOTU [Rok-Msc-Dzien]");
+        dataWylotu.setPrefColumnCount(17);
+        dataWylotu.setLayoutX(40);
+        dataWylotu.setLayoutY(270);
+        dataWylotu.setVisible(true);
+        dataWylotu.setOnAction(new EventHandler<ActionEvent>() {

```

Comment:
Zamiast rozwlekac tutaj kod i implementowac metody interfejsu w metodzie setOnAction to mozna stworzyc osobna klase np `TextFieldHandler`, ktora bedzie implementowac interfejs `EventHandler<ActionEvent>` i w tamtej klasie nadpisac metode handle i pozniej tylko przekazac sam obiekt do tej metody.  Jak pozniej zauwazylem wiele razy pojawia sie ten interfejs i ich implementacje nie wiele sie roznia. Mozna poglowkowac zeby stworzyc maksymalnie ze 2 klasy, ktore beda go implementowaly i tak manipulowaly danymi zeby nie tworzyc tych klas 10 tylko mniej ale zeby byly elastyczne